src/tables.js

```diff
@@ -9,6 +9,7 @@ const Flow = {
   STEM_HEIGHT: 32,
   STAVE_LINE_THICKNESS: 2,
   RESOLUTION: 16384,
+  INITIAL_ARTICULATION_OFFSET: -0.5,
```

Comment:
I'd like to only put things here that are used across multiple classes, otherwise keep them along with the class. It makes things more cohesive, simpler to maintain, and less likely to rot.

Also, I think it's cleaner to say `VF.Articulation.INITIAL_OFFSET = 1` than `VF.INITIAL_ARTICULATION_OFFSET = 1`. Same for the other values here, but I just haven't had a chance to clean them up.